### PR TITLE
Fix docs rendering

### PR DESCRIPTION
**Pull request by jezdez**
This fixes the build process, as visible (in the now enabled pull request build) here: https://django-dbtemplates--127.org.readthedocs.build/en/127/

### CODE REVIEW

File: requirements/docs.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+django
```
Comment (mpasternak) [inline requirements/docs.txt:1]: 👍 for the separate requirements to build the docs, from my experience it is not only less burden for RTD service but some build requirements can break the docs actually.

**Review by mpasternak — APPROVED** [review]
OK by me

**Comment by mpasternak** [comment]
Merged, thanks.

File: requirements/docs.txt (hunk 1 above)
Comment (jezdez) [reply requirements/docs.txt:1]: Yep!

**Pull request merged**